src/Tracker.ts

```diff
@@ -100,6 +112,77 @@ export class Tracker {
     queryMethods.forEach((method) => (this.history[method].length = 0));
   }
 
+  public resetTransactions() {
+    this.transactions.clear();
+  }
+
+  public getTransactions(): TransactionState[] {
```

Comment:
I think that it should be on the `history` object, WDYT?
Instead of a public method on the Tracker

Reply:
Sure thing